frontend/src/components/transactions/ActionMessage.tsx

```diff
@@ -253,9 +205,8 @@ const ActionMessage = (props: Props<AnyAction>) => {
       </>
     );
   }
-  return (
-    <MessageRenderer {...(props as any)} showDetails={props.showDetails} />
```

Comment:
Why did we have this extra `showDetails` double application?

Reply:
Have we ever use it anywhere in the code? I don’t remember...

Reply:
I think it is some sort of a refactoring artifact